aws/table_aws_eks_identity_provider_config.go

```diff
@@ -148,21 +147,24 @@ func listEksIdentityProviderConfigs(ctx context.Context, d *plugin.QueryData, h
 		return nil, err
 	}
 
+	// As per the API document input parameter MaxResults should support the value of 100.
+	// However with value of 100, API is throwing an error - InvalidParameterException: maxResults needs to be 1.
+	// Raised an issue with AWS SDK - https://github.com/aws/aws-sdk-go/issues/4457
 	param := &eks.ListIdentityProviderConfigsInput{
 		ClusterName: cluster.Name,
-		MaxResults:  aws.Int64(100),
+		//MaxResults:  aws.Int64(100),
```

Comment:
Do any other values beside 1 work for `MaxResults`? Keeping it as 1 will definitely create a lot of requests for list queries.